manuscript/revision_response.Rmd

```diff
@@ -0,0 +1,118 @@
+---
+output: pdf_document
+---
+
+Revision Round 1 Notes, Deep RL for Conservation Decisions
+
+Reviewer 1:
+
+1) “I just noticed very few typos: Line 132: “forgo”, and line 318: “out-preforms”.”
+
+We have fixed the line 318 typo. Use of "forgo" on line 132 was intentional.
+
+2) As a computer scientist, I am just curious to know more about the hardware and software used for the experiments, especially the GPU used and how much time did it take to converge? At least the order of magnitude (hours or days)?
+
+All algorithms were run on an NVIDIA Quadro RTX 8000 GPU. The training  budget for the fishing scenario was 300K timesteps (3K runs, taking about 1 hr). Tipping point training budget was 3M timesteps (6K runs, taking several hours). Precise run times vary with machine load and algorithm. Software details and hyperparameters are provided in the associated GitHub repo, https://github.com/boettiger-lab/rl-intro/. Hyperparameter tuning typically required 100s of training runs using both Optuna and manual adjustments.
```

Comment:
ok just figured I should actually benchmark instead of going from memory.  The fishing script https://github.com/boettiger-lab/rl-intro/blob/main/python/fishing_TD3_tuned.py takes 25 minutes on the Quadro (cirrus), but the tipping point case takes longer; https://github.com/boettiger-lab/rl-intro/blob/main/python/conservation_TD3.py is at 3 hrs and still running.  I'll ping back when it's done so we can be a bit more precise